projects/1.18/assets/chileancraft/chileancraft/lang/zh_cn.json

```
@@ -0,0 +1,871 @@
+{
+  "advancements.alabordaje.descr": "获得一柄水手的佩剑",
+  "advancements.alabordaje.title": "上船吧，孩子！",
```

```suggestion
  "advancements.alabordaje.title": "孩子们，登船！",
```
https://colegioingles.cl/arturo-prat-combate-naval-de-iquique-en-1879/
https://en.wikipedia.org/wiki/Arturo_Prat

该语句来源于智利将领Arturo Prat于硝石战争（War of the Pacific）中的遗言。

此外，你可能需要一本[西语词典](https://www.esdict.cn/)。西语和英语有很多近似的地方，例如原文的`muchachos`是`muchacho`的复数形式。

---

啊 ecsidt的话翻译的时候我倒是一直在用的